cogs/utils/paginator.py

```diff
@@ -0,0 +1,302 @@
+import asyncio
+import discord
+from discord.ext.commands import Paginator as CommandPaginator
+
+FIRST_EMOJI = "\u23EE"   # [:track_previous:]
+LEFT_EMOJI = "\u2B05"    # [:arrow_left:]
+RIGHT_EMOJI = "\u27A1"   # [:arrow_right:]
+LAST_EMOJI = "\u23ED"    # [:track_next:]
+PAGE_NUM_EMOJI = ":1234: "
+STOP_EMOJI = ":stop_button: "
+INFO_EMOJI = ":information_source: "
+
+class CannotPaginate(Exception):
+    pass
+
+
+class Pages:
+    """
+    Implements a paginator that queries the user for the
+    pagination interface.
+    Pages are 1-index based, not 0-index based.
+    If the user does not reply within 2 minutes then the pagination
+    interface exits automatically.
+    Parameters
+    ------------
+    ctx: Context
+        The context of the command.
+    entries: List[str]
+        A list of entries to paginate.
+    per_page: int
+        How many entries show up per page.
+    show_entry_count: bool
+        Whether to show an entry count in the footer.
+    Attributes
+    -----------
+    embed: discord.Embed
+        The embed object that is being used to send pagination info.
+        Feel free to modify this externally. Only the description,
+        footer fields, and colour are internally modified.
+    permissions: discord.Permissions
+        Our permissions for the channel.
+    """
```

Comment:
Can we remove this comment, or make it smaller?

Reply:
That should be a part of #11 don't change the comment here.